bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -142,22 +134,26 @@ def detailed_perf_stats(config: WTPerfConfig, perf_stats: PerfStatCollection):
     return as_dict
 
 
-def run_test_wrapper(config: WTPerfConfig, operations: List[str] = None, argument: str = None):
+def run_test_wrapper(config: WTPerfConfig, operations: List[str] = None, arguments: List[str] = None):
     for test_run in range(config.run_max):
         print("Starting test  {}".format(test_run))
-        run_test(config=config, test_run=test_run, operations=operations, argument=argument)
+        run_test(config=config, test_run=test_run, operations=operations, arguments=arguments)
         print("Completed test {}".format(test_run))
 
 
-def run_test(config: WTPerfConfig, test_run: int, operations: List[str] = None, argument: str = None):
+def run_test(config: WTPerfConfig, test_run: int, operations: List[str] = None, arguments: List[str] = None):
     test_home = create_test_home_path(home=config.home_dir, test_run=test_run, operations=operations)
     command_line = construct_wtperf_command_line(
         wtperf=config.wtperf_path,
         env=config.environment,
-        argument=argument,
+        arguments=arguments,
         test=config.test,
         home=test_home)
-    subprocess.run(command_line, check=True)
+    try:
+        subprocess.run(command_line, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE, universal_newlines=True)
```

Comment:
These new arguments allow us to print stderr as plaintext below.
If we can guarantee a minimum python version of 3.7 (The VMs have 3.6.9) then this can be simplified to `capture_output=True, text=True`